src/pages/signin/SignInPageLayout/SignInPageContent.js

```diff
@@ -55,7 +55,8 @@ const SignInPageContent = props => (
             <LoginKeyboardAvoidingView
                 behavior="position"
                 contentContainerStyle={[
-                    props.isSmallScreenWidth ? styles.signInPageNarrowContentMargin : styles.signInPageWideLeftContentMargin,
+                    props.isSmallScreenWidth && styles.signInPageNarrowContentMargin,
```

Comment:
When `props.isSmallScreenWidth` is falsy, this would end up with an array with a `null` or `false` element, is that allowed? (same below)